src/main/scala/org/galaxio/gatling/kafka/actions/KafkaRequestAvro4sActionBuilder.scala

```diff
@@ -10,8 +11,11 @@ import org.galaxio.gatling.kafka.protocol.KafkaProtocol
 import org.galaxio.gatling.kafka.request.builder.Avro4sAttributes
 
 import scala.jdk.CollectionConverters._
+import scala.reflect.ClassTag
+
+case class KafkaRequestAvro4sActionBuilder[K: ClassTag, V: ClassTag](attributes: Avro4sAttributes[K, V])
```

Comment:
Why do you use `ClassTag` here?

Reply:
I did it like there was in KafkaRequestReply